templates/PlayFab_Api.cpp.ejs

```diff
@@ -38,13 +38,13 @@ namespace PlayFab
     {
 <%- getRequestActions("        ", apiCall, false) %>
         IPlayFabHttpPlugin& http = *PlayFabPluginManager::GetPlugin<IPlayFabHttpPlugin>(PlayFabPluginContract::PlayFab_Transport);
-        const auto requestJson = request.ToJson();
+        const auto& requestJson = request.ToJson();
```

Comment:
>auto [](start = 14, length = 4)

left comments elsewhere that I think should also be reflected in this template file

this use of auto should be an explicit type

the use of unique_ptr on 47 could be auto

internalPtr on 71 should be an explicit type